packages/react/src/themes/teams/components/Popup/popupStyles.ts

```diff
@@ -1,16 +1,34 @@
+import * as React from 'react'
 import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
 import { PopupProps } from '../../../../components/Popup/Popup'
 import { PopupVariables } from './popupVariables'
 
 const popupStyles: ComponentSlotStylesInput<PopupProps, PopupVariables> = {
   root: (): ICSSInJSStyle => ({}),
 
-  popup: ({ variables: v }): ICSSInJSStyle => ({
+  popup: ({ props: p, variables: v }): ICSSInJSStyle => ({
     zIndex: v.zIndex,
     position: 'absolute',
     textAlign: 'left',
-    color: v.contentColor,
-    background: v.contentBackgroundColor,
+
+    /*
+     * This fix handles all cases with wrapped focus trap.
+     */
+    ...(React.isValidElement(p.content) && {
```

Comment:
actually, this should also apply in case if render function is provided as an argument :( as in that case slot component will be also replaced. So, I think that we may just check that this is not an object or primitive